.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,87 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+name: 🐛 Bug Report
+description: "Problems with the Ultralytics YOLO Flutter plugin"
+labels: [bug, triage]
+type: "bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an Ultralytics YOLO Flutter Plugin 🐛 Bug Report!
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please search the Ultralytics YOLO Flutter Plugin [docs](https://github.com/ultralytics/yolo-flutter-app/tree/main/doc) and [issues](https://github.com/ultralytics/yolo-flutter-app/issues) to see if a similar bug report already exists.
+      options:
+        - label: >
+            I have searched https://github.com/ultralytics/yolo-flutter-app/issues and did not find a similar report.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Project area
+      description: |
+        Help us route the report to the right maintainers.
+      multiple: true
+      options:
+        - "Plugin API"
+        - "Android build"
+        - "iOS build"
+        - "Sample app/example"
+        - "Documentation"
+        - "Other"
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Bug
+      description: Please describe the issue in detail so we can reproduce it in Ultralytics YOLO Flutter Plugin. Include logs, screenshots, console output, and any context that helps explain the problem.
+      placeholder: |
+        💡 ProTip! Include as much information as possible (logs, tracebacks, screenshots, etc.) to receive the most helpful response.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Environment
```

💭 **SUGGESTION**: The Environment section is good, but you could make it even more actionable for this repo by explicitly calling out Flutter- and mobile-specific details (e.g., Flutter version, Dart version, Android/iOS SDK versions, device model). This will help triage mobile build/runtime issues faster.

**Suggested change:**
```suggestion
      description: Share the platform, Flutter, and device version information relevant to your report.
```